_activities/Lifebeyondtheborders/002.md

```diff
@@ -0,0 +1,214 @@
+---
```

Comment:
Can we change the folder name from camelcase to dash-separated ?

Reply:
I've changed the styling of the folder name...

Reply:
it still looks like camel case. we need the url to be like dash-sparated-lower-cased

Reply:
I've opened a new pull request. Updated the links in that.